rpm/build-config

```diff
@@ -1,46 +1,643 @@
+#CONFIG_DRIVER_MACSEC_QCA=y
+
+# Driver interface for Linux MACsec drivers
+# Disabled because it needs MACsec from 4.6 read commit below in kernel: 
+# dece8d2b78d19df7fe5e4e965f1f0d1a3e188d1b
+#CONFIG_DRIVER_MACSEC_LINUX=y
+
+# Driver interface for the Broadcom RoboSwitch family
+#CONFIG_DRIVER_ROBOSWITCH=y
+
+# Driver interface for no driver (e.g., WPS ER only)
+#CONFIG_DRIVER_NONE=y
+
+# Solaris libraries
+#LIBS += -lsocket -ldlpi -lnsl
+#LIBS_c += -lsocket
+
+# Enable IEEE 802.1X Supplicant (automatically included if any EAP method or
+# MACsec is included)
 CONFIG_IEEE8021X_EAPOL=y
+
+# EAP-MD5
 CONFIG_EAP_MD5=y
+
+# EAP-MSCHAPv2
 CONFIG_EAP_MSCHAPV2=y
+
+# EAP-TLS
 CONFIG_EAP_TLS=y
+
+# EAL-PEAP
 CONFIG_EAP_PEAP=y
+
+# EAP-TTLS
 CONFIG_EAP_TTLS=y
+
+# EAP-FAST
 CONFIG_EAP_FAST=y
+
+# EAP-TEAP
+# Note: The current EAP-TEAP implementation is experimental and should not be
+# enabled for production use. The IETF RFC 7170 that defines EAP-TEAP has number
+# of conflicting statements and missing details and the implementation has
+# vendor specific workarounds for those and as such, may not interoperate with
+# any other implementation. This should not be used for anything else than
+# experimentation and interoperability testing until those issues has been
+# resolved.
+#CONFIG_EAP_TEAP=y
+
+# EAP-GTC
 CONFIG_EAP_GTC=y
+
+# EAP-OTP
 CONFIG_EAP_OTP=y
-CONFIG_EAP_AKA=y
+
+# EAP-SIM (enable CONFIG_PCSC, if EAP-SIM is used)
+#CONFIG_EAP_SIM=y
+
+# Enable SIM simulator (Milenage) for EAP-SIM
+#CONFIG_SIM_SIMULATOR=y
+
+# EAP-PSK (experimental; this is _not_ needed for WPA-PSK)
+#CONFIG_EAP_PSK=y
+
+# EAP-pwd (secure authentication using only a password)
+CONFIG_EAP_PWD=y
+
+# EAP-PAX
 CONFIG_EAP_PAX=y
+
+# LEAP
 CONFIG_EAP_LEAP=y
+
+# EAP-AKA (enable CONFIG_PCSC, if EAP-AKA is used)
+CONFIG_EAP_AKA=y
+
+# EAP-AKA' (enable CONFIG_PCSC, if EAP-AKA' is used).
+# This requires CONFIG_EAP_AKA to be enabled, too.
+#CONFIG_EAP_AKA_PRIME=y
+
+# Enable USIM simulator (Milenage) for EAP-AKA
+#CONFIG_USIM_SIMULATOR=y
+
+# EAP-SAKE
 CONFIG_EAP_SAKE=y
+
+# EAP-GPSK
 CONFIG_EAP_GPSK=y
+# Include support for optional SHA256 cipher suite in EAP-GPSK
 CONFIG_EAP_GPSK_SHA256=y
+
+# EAP-TNC and related Trusted Network Connect support (experimental)
 CONFIG_EAP_TNC=y
+
+# Wi-Fi Protected Setup (WPS)
 CONFIG_WPS=y
-CONFIG_AP=y
+# Enable WPS external registrar functionality
+#CONFIG_WPS_ER=y
+# Disable credentials for an open network by default when acting as a WPS
+# registrar.
+#CONFIG_WPS_REG_DISABLE_OPEN=y
+# Enable WPS support with NFC config method
+#CONFIG_WPS_NFC=y
+
+# EAP-IKEv2
 CONFIG_EAP_IKEV2=y
+
+# EAP-EKE
+#CONFIG_EAP_EKE=y
+
+# MACsec
+CONFIG_MACSEC=y
+
+# PKCS#12 (PFX) support (used to read private key and certificate file from
+# a file that usually has extension .p12 or .pfx)
 CONFIG_PKCS12=y
+
+# Smartcard support (i.e., private key on a smartcard), e.g., with openssl
+# engine.
 CONFIG_SMARTCARD=y
-CONFIG_DEBUG_FILE=y
```

Comment:
Got removed. On purpose?

Reply:
Yes wpa_supplicant should log to syslog.

Reply:
Then again this shouldn't be where the log is written by default but what is supported as log target?

Reply:
Support either stderr, syslog or file.

Reply:
Don't think it's a choice between things. Default seems to be file and syslog.

Reply:
I haven't seen even that we'd use direct separate file logging with this at any step. Usually this has been working quite well and the amount of logs that are in journal have been sufficient. I think it is better to log everything only to journal (syslog) so other interactions with other components, starting from kernel from up to, e.g., ConnMan are on the same time sequence.

Reply:
Yeah and you can also use journalctl -u to limit the amount of logs you want to get, e.g. journal -u user@10000 or journalctl -u connman -u ofono -f.